build-python.sh

```diff
@@ -352,7 +354,7 @@ if test "${BUILD_WHEEL}" = true; then
     python -m build \
         --wheel \
         --outdir ../dist \
-        ${BUILD_ARGS} \
+        $BUILD_ARGS \
```

Comment:
This raises spellcheck info level `SC2086 (info): Double quote to prevent globbing and word splitting.` but adding the double quotes breaks the command (BUILD_ARGS is empty) I don't understand the root cause so I leave it as it. This is repeated twice in the file.